requirements.txt

```diff
@@ -26,6 +26,7 @@ docutils==0.16
 entrypoints==0.3
 flake8==3.9.0
 flatbuffers==1.12
+foolbox==3.3.1
```

Comment:
The toolbox should also be added in ``setup.cfg`` -> ``install_requires``.

Reply:
Done in 3dde9f4